Documentation/ApiOverview/Backend/BroadcastChannels.rst

```diff
@@ -29,26 +38,30 @@ component name and the event name, e.g. `typo3:my_extension:my_event`.
 
 .. attention::
 
-   Since a polyfill is in place to add support for Microsoft Edge, the payload must contain JSON-serializable content
-   only.
-
+    Since a polyfill is in place to add support for Microsoft Edge, the
```

Comment:
This admonition can be deleted. The polyfill was assumably for older Edge versions which are not Chromium-based. The polyfill was removed with commit https://github.com/TYPO3/typo3/commit/47481cbdd8779f15d69095ef479156884e522929#diff-ce8f178fabf4fc619e29c4a9a6caabdc90f8769fc10d631dd1aba54d24801349.